managed/src/test/java/com/yugabyte/yw/controllers/BackupsControllerTest.java

```diff
@@ -282,6 +283,8 @@ public void testDeleteBackup() {
     Result result = deleteBackup(resultNode, null);
     assertEquals(200, result.status());
     JsonNode json = Json.parse(contentAsString(result));
+    CustomerTask customerTask = CustomerTask.findByTaskUUID(fakeTaskUUID);
+    assertEquals(customerTask.getTargetUUID(), backup.getBackupInfo().universeUUID);
     assertEquals(json.get("taskUUID").size(), 1);
     assertAuditEntry(1, defaultCustomer.uuid);
   }
```

Comment:
please add migration file to migrate old data.

Reply:
Was it in this state for some time already? Then yes. We should migrate.